jgrapht-core/src/main/java/org/jgrapht/intervalgraph/IntervalGraph.java

```diff
@@ -0,0 +1,14 @@
+package org.jgrapht.intervalgraph;
+
+import org.jgrapht.Graph;
+import org.jgrapht.intervalgraph.interval.Interval;
+
+import java.util.Collection;
+
+public interface IntervalGraph<T extends Comparable<T>> {
```

Comment:
Do we really need an interface or would just an implementation that implements the JGraphT graph interface be enough?